test/unit/config.spec.ts

```diff
@@ -33,28 +35,33 @@ describeCommand('Config data', () => {
       },
       {
         data: { beeApiUrl, beeDebugApiUrl },
+        message: 'Config error: `ensNetwork` is not defined or empty',
+      },
+      {
+        data: { beeApiUrl, beeDebugApiUrl, ensNetwork },
         message: 'Config error: `accounts` is not an object',
       },
       {
-        data: { beeApiUrl, beeDebugApiUrl, accounts: 'hello' },
+        data: { beeApiUrl, beeDebugApiUrl, ensNetwork, accounts: 'hello' },
```

Comment:
you could rewrite these test records in a way to not modify it after every config structure change in this way
```suggestion
        data: { ...correctConfig, accounts: 'hello' },
```

Reply:
the absence of each parameter checks in turn. If a valid object is passed, the validation will not work as intended.